lang/resources/org/partiql/type-domains/partiql.ion

```diff
@@ -636,6 +636,26 @@ may then be further optimized by selecting better implementations of each operat
         (exclude let_binding)
         (include (product let_binding value::expr decl::var_decl))
 
+        // Inputs to Aggregations
+        (include
+            (product group_key
+                     expr::expr
+                     as_var::var_decl
+            )
+            (product group_key_list
+                     keys::(* group_key 1)
+            )
+            (product aggregate_function
+                quantifier::set_quantifier
+                name::symbol
+                arg::expr
+                as_vars::(* var_decl 1)
+            )
+            (product aggregate_function_list
+                functions::(* aggregate_function 1)
+            )
```

Comment:
Can we remove this so we don't need the nested `functions.functions`

Reply:
Modified so that (aggregate) uses `functionList` instead to keep it consistent with group keys. LMK if that's sufficient. Now, it should be `functionList.functions`